Contribute/how-to-write-links.md

```diff
@@ -12,7 +12,7 @@ ms.date: 10/31/2018
 
 This article describes how to use hyperlinks from pages hosted at docs.microsoft.com. Links are easy to add into markdown with a few varying conventions. Links point users to content in the same page, in other neighboring pages, or on external sites and URLs.
 
-The docs.microsoft.com site backend uses Open Publishing Services (OPS), which supports [CommonMark](https://commonmark.org/)-compliant markdown parsed through the [Markdig](https://github.com/lunet-io/markdig) parsing engine. This markdown flavor is mostly compatible with [GitHub Flavored Markdown (GFM)](https://help.github.com/categories/writing-on-github/), as most docs are stored in GitHub and can be edited there. Additional functionality is added through Markdown extensions.
+The docs.microsoft.com site backend uses Open Publishing Services (OPS), which supports [CommonMark][]-compliant markdown parsed through the [Markdig][] parsing engine. This markdown flavor is mostly compatible with [GitHub Flavored Markdown (GFM)][GFM], as most docs are stored in GitHub and can be edited there. Additional functionality is added through Markdown extensions.
```

Comment:
I think you didn't complete the new links. I see 6 instances of this in the doc.

Reply:
Those are reference-style links. They are defined at the bottom of the markdown file.

Reply:
See the preview https://review.docs.microsoft.com/en-us/contribute/how-to-write-links?branch=pr-en-us-373

The links are valid

Reply:
I see, I didn't know you could leave the brackets empty like that.